app/routes/core-data/admin-unit.js

```diff
@@ -108,7 +108,15 @@ export default class AdminUnitRoute extends Route {
           return (await scope.locatedWithin).label;
         })()
       : null;
-
+    const changeEvents = [...(await adminUnit.changedBy)]
+    let isCity = false;
```

Comment:
When isCity is true break out of loop.
Or use a more functional approach maybe?
```Javascript
const eventTypePromises = changeEvents.map((event)=>event.type);
const isCity = (await Promise.all(eventTypePromises)).some((eventType)=>eventType.id==='e4c3d1ef-a34d-43b0-a18c-f4e60e2c8af3');
```

Reply:
true, I added the code from OP and didn't double check, I added a break